apps/condo/domains/news/utils/clientSchema/NewsRenders.tsx

```diff
@@ -80,19 +82,20 @@ const POSTFIX_PROPS: TextProps = { type: 'secondary', style: { whiteSpace: 'pre-
 
 export const ResendNewsButton = ({ intl, newsItem }) => {
     const ResendMessage = intl.formatMessage({ id: 'pages.condo.news.resend' })
-
-    //TODO(DOMA-5917) this functionality will be implemented as part of next task
+    
     const handleClick = useCallback((e) => {
         e.stopPropagation()
     }, [])
     
```

Comment:
this icon is in a table row and clicking on it is perceived as clicking on it and on the table row at the same time and I need to block capturing phase so that clicking on the table row is not done